src/org/scada_lts/utils/SystemSettingsUtils.java

```diff
@@ -762,4 +765,24 @@ public static int getValidationSearchCyclicDepth() {
             return defaultValue;
         }
     }
+
+    public static int getDefaultPurgePeriod() {
+        try {
+            String period = ScadaConfig.getInstance().getConf().getProperty(DEFAULT_PURGE_PERIOD, "1");
+            return Integer.parseInt(period);
+        } catch (Exception e) {
+            LOG.error(e.getMessage());
+            return 0;
+        }
+    }
+
+    public static int getDefaultPurgePeriodType() {
+        try {
+            String periodType = ScadaConfig.getInstance().getConf().getProperty(DEFAULT_PURGE_PERIOD_TYPE, "7");
+            return Integer.parseInt(periodType);
+        } catch (Exception e) {
+            LOG.error(e.getMessage());
+            return 7;
+        }
+    }
```

Comment:
Same as before, you need to use the pattern that is used in this class for the default value.